--- InvenTree/InvenTree/settings.py.orig	2024-03-02 22:34:18.794905000 -0400
+++ InvenTree/InvenTree/settings.py	2024-03-02 22:34:34.394520000 -0400
@@ -764,9 +764,6 @@
                 socket.TCP_KEEPCNT: int(os.getenv('CACHE_KEEPALIVES_COUNT', '5')),
                 socket.TCP_KEEPIDLE: int(os.getenv('CACHE_KEEPALIVES_IDLE', '1')),
                 socket.TCP_KEEPINTVL: int(os.getenv('CACHE_KEEPALIVES_INTERVAL', '1')),
-                socket.TCP_USER_TIMEOUT: int(
-                    os.getenv('CACHE_TCP_USER_TIMEOUT', '1000')
-                ),
             },
         },
     }
